src/Event/ImageProcessingListener.php

```diff
@@ -77,6 +77,12 @@ public function implementedEvents() {
 	 * @return boolean
 	 */
 	protected function _autoRotate($imageFile, $format) {
+		// Auto-rotate applies only to JPEG files (i.e. not to PNG, GIF, etc..)
```

Comment:
This comment is now wrong, it's jpeg and tiff. Also please move that into the doc block and keep a new line between the first line and the params in the doc block.

I'll merge this PR after these two changes are done.